### DISCUSSION TITLE
GPU support

### DISCUSSION THREAD

**Discussion by hcadavid** [vantage6/v6-on-kubernetes-PoC#3] [Q&A] [answered]
Based on v6's DockerManager [source code](https://github.com/vantage6/vantage6/blob/fdaf7c3dcb653f89fd74f3ebb4d2cce3c80516ef/vantage6-node/vantage6/node/docker/docker_manager.py#L227), it seems that it is possible to enable the use of GPU resources for the containerised algorithms. 

However, GPU support is something that has been discussed (in other scenarios) as a feature yet to be implemented. @bartvanb, as this is something to be explored on this PoC, could you give us some details on to what extent is this working?

**Comment by bartvanb** [comment]
There is the option to enable GPU access to an algorithm container, see https://github.com/vantage6/vantage6/issues/597 and links to the PR therein. It is my understanding that in order to make use of this, the node has to run on (the machine with) the GPU itself though.

The open issue that is relevant here is I think this one: https://github.com/vantage6/vantage6/issues/431, which asks to run algorithms non-locally

**Reply by hcadavid** [reply, marked as answer]
Thanks @bartvanb for the clarification!
